int16                            type
geometry_msgs/PoseWithCovariance pose
